src/applications/personalization/profile/components/personal-information/addresses/AddressesTable.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import AddressField from './AddressField';
+import ContactInformationField from '~/applications/personalization/profile/components/personal-information/ContactInformationField';
```

Comment:
I know we don't really have guidance for when to use the Babel aliases or not, but I think `from '../ContactInformationField'` would work here?